build_tools/compiler/play_processor.rb

```diff
@@ -12,7 +12,7 @@ class PlayProcessor < TemplateProcessor
       # top_of_page has a special purpose: it is required by Play to define the
       # parameters to pass when rendering
       # https://www.playframework.com/documentation/2.2.x/ScalaTemplates#Template-parameters
-      top_of_page: '@(title: Option[String], bodyClasses: Option[String], htmlLang: Option[String] = None)(head:Html, bodyStart:Html, bodyEnd:Html, insideHeader:Html, afterHeader:Html, footerTop:Html, footerLinks:Html, headerClass:Html = Html.empty, propositionHeader:Html = Html.empty, homepageUrl:Html = Html.empty, globalHeaderText:Html = Html.empty, cookieMessage:Html = Html.empty)(content:Html)',
+      top_of_page: '@(title: Option[String], bodyClasses: Option[String], htmlLang: Option[String] = None)(head:Html, bodyStart:Html, bodyEnd:Html, insideHeader:Html, afterHeader:Html, footerTop:Html, footerLinks:Html, headerClass:Html = Html.empty, propositionHeader:Html = Html.empty, homepageUrl:Html = Html.empty, globalHeaderText:Html = Html.empty, cookieMessage:Html = Html.empty, skipLinkMessage:String, logoLinkTitle:String, licenceMessage:Html, crownCopyrightMessage:String)(content:Html)',
```

Comment:
@froddd 

> Is this in respects to the HTML entity in the default string? 

Yes.

> Will this affect any value passed as the variable, and if so, should we extend the reasoning to the other strings, to allow for special characters?

I based my comment on the other parameters. I'm afraid I don't know enough about Play templates to say with any confidence - maybe @HolyHaddock could explain?